### PR TITLE
feature(point): option to link OrientedImageMaterial to PointsMaterial.

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
Add option to link `OrientedImageMaterial` to `PointsMaterial` when creating Points Layer.
I'll add an example when i'll have the datas

### CODE REVIEW

File: src/Renderer/Shader/PointsFS.glsl [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
-    gl_FragColor = vColor;
+#if defined(USE_TEXTURES_PROJECTIVE)
+    if (!pickingMode) {
+        vec4 color = vColor;
+        vec4 aColor;
+        float alpha = 0.0;
 
-    #include <logdepthbuf_fragment>
+        #pragma unroll_loop
+        for (int i = 0; i < ORIENTED_IMAGES_COUNT; i++) {
```
Comment (zarov) [inline src/Renderer/Shader/PointsFS.glsl:20]: Maybe factorize this part with `ProjectiveTextureFS.glsl` ? It is almost the same, beside the `#ifdef USE_BASE_MATERIAL`.

File: src/Renderer/PointsMaterial.js [hunk 2]
```diff
@@ -42,9 +60,23 @@ class PointsMaterial extends RawShaderMaterial {
         if (__DEBUG__) {
             this.defines.DEBUG = 1;
         }
+
         this.updateUniforms();
     }
 
+    copy(source) {
+        super.copy(source);
+        if (source.uniforms.projectiveTextureAlphaBorder) {
```
Comment (zarov) [inline src/Renderer/PointsMaterial.js:70]: Why would you need to do that ? This should be handled by the implementation of `ShaderMaterial.copy`.

File: src/Renderer/Shader/PointsFS.glsl (hunk 1 above)
Comment (gchoqueux) [reply src/Renderer/Shader/PointsFS.glsl:20]: Factorized

File: src/Renderer/PointsMaterial.js (hunk 2 above)
Comment (gchoqueux) [reply src/Renderer/PointsMaterial.js:70]: Because, it's a link material to oriented image material, it's necessary to reference these uniforms.

Comment (zarov) [reply src/Renderer/PointsMaterial.js:70]: My bad, I understand what you are doing here !

**Review by zarov — APPROVED** [review]

**Pull request merged**